### PR TITLE
Add nodejs test workflow

### PR DESCRIPTION
**Pull request by greatbacon**
I'm experimenting with github workflows!

### CODE REVIEW

**Comment by ryanatkn** [comment]
👀

File: .github/workflows/nodejs.yml [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+    branches: [ master ]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        node-version: [12.16]
```
Comment (ryanatkn) [inline .github/workflows/nodejs.yml:18]: Neat I've never used these. See PR #8 about a version bump.

**Comment by ryanatkn** [comment]
Want to merge or do you have more changes queued up? `npm run test` obviously does nothing yet but `npm run build` seems like it gives us quite a lot of value already.

I know I'm late to the party on CI but this is pretty cool. Was looking at how Svelte builds on [Ubuntu/Mac/Windows](https://github.com/sveltejs/svelte/blob/master/.github/workflows/ci.yml#L9).

Down the road we could add Prettier's `--check` to this. Right now though we'd want to exclude `.svelte` files because of [this bug](https://github.com/UnwrittenFun/prettier-plugin-svelte/issues/24) that affects a small number of files - maybe we just wait. Also I haven't used ESLint in a while because TypeScript+Prettier is so helpful, but I imagine we'll want to use at least some of its checks eventually. I really like the idea of using CI over git hooks for these workflows.

**Comment by greatbacon** [comment]
@ryanatkn yeah just wanted to wait for an official approval ;P  CI is dope, and I'm glad that we're able to integrate it without too much fuss. And yeah, once we have it config, we can add a prettier config later probably even in the same PR as the seetup.

**Pull request merged**